src/components/toolbox/dialog/dialog.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DialogHolder from './holder';
+import Title from './title';
+import Description from './description';
+import Options from './options';
+import styles from './dialog.css';
+
+const Dialog = ({ children, hasClose }) => (
+  <div className={styles.wrapper}>
+    {hasClose && (
+      <span
+        onClick={DialogHolder.hideDialog}
+        className={styles.closeBtn}
+      />
+    )}
+    {children}
+  </div>
+);
+
+Dialog.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element,
+    PropTypes.arrayOf(PropTypes.element),
+  ]).isRequired,
+  hasClose: PropTypes.bool,
+};
+
+Dialog.defaultProps = {
+  hasClose: false,
+};
+
+Dialog.displayName = 'Dialog';
```

Comment:
Unfortunately, the same approach won't work in the sub-components (e.g. `Dialog.Title`)

Reply:
Actually I think that this line is not necessary in this component, since it follows the approach of implicit naming. it will appear as `<Dialog>` on the React tree